microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -5,147 +5,221 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 08/18/2023
+ms.date: 12/14/2023
 ---
 # Microsoft Edge Add-ons store developer policies
```

Comment:
```suggestion
Developer policies for the Microsoft Edge Add-ons store 
```
start w/ key words; avoid a long string of modifiers followed by the key words.  easier/quicker to see which article is meant

Reply:
@Bhuvana-Priya (@AbhisekhMSFT) - I changed the title to start with "Developer policies".

From the POV of a legal document's title, considered in isolation, one might lean towards:
Microsoft Edge Add-ons store developer policies
(stiff & formal sounding)

But from the POV of looking at article titles in the TOC, lean towards:
Developer policies for the Microsoft Edge Add-ons store
to help quickly differentiate & characterize the various articles.  Practical.

Reply:
The repo uses too many different phrases:
```
The _Microsoft Edge Add-ons store_<!--36--> is also called 
the _Edge Add-ons store_,<!--12--> 
the _Microsoft Edge Add-ons website_,<!--89--> or 
the _Edge Add-ons website_.<!--5-->
```
That's covered by work item 47150845 - [Extensions] Normalize name of store/website, and "add-ons" vs. "extensions"